src/main/java/WayofTime/alchemicalWizardry/AlchemicalWizardry.java

```diff
@@ -110,15 +111,23 @@
 	public static boolean parseTextFiles = false;
 
     public static boolean doMeteorsDestroyBlocks = true;
-    public static String[] diamondMeteorArray;
-    public static int diamondMeteorRadius;
-    public static String[] stoneMeteorArray;
-    public static int stoneMeteorRadius;
-    public static String[] ironBlockMeteorArray;
-    public static int ironBlockMeteorRadius;
-    public static String[] netherStarMeteorArray;
-    public static int netherStarMeteorRadius;
-
+    public static String[] Meteor00Array;
```

Comment:
That is so horrible. Why not have a structure like "MeteorDescription" or something, and a map: id -> description
(This is not comment about your code, but about original of course)

Reply:
Ok. I don't even know what a structure or map is, even if you wanted me to change it, so I guess that works out?

Reply:
Your changes as they are are ok, it is just the original code was not designed to be extensible. No wonder 1.12 is incompatible to the extent of being useless, author probably rewritten it from scratch

Reply:
Is there anything I can realistically do here? I want to add a whole bunch of meteors, but I don't want to recode the whole thing. Or rather I don't know how. Is there any problem just adding a couple dozen meteors code/lagwise?

Reply:
Um, perhaps I'll refactor that shit a bit and you apply your changes later?

Reply:
OK